### DISCUSSION TITLE
Project interactions

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#20] [Announcements]
Dear All,

I add an additional task so that different team members have access to all project repos (e0f2e1e). See the [project folder](https://github.com/STAT5125-UConn/2022Spring/tree/main/project) for more details. 

It is to your benefit to have your project materials updated as soon as possible.

**Comment by Ossifragus** [comment]
Deadlines for project interactions (f4412cb):

    Step 1: give other students access to your project repo by 8 PM <2022-04-19 Tue>
    Step 2: send your filled ProjectScore to haiying.wang@uconn.edu by 8 PM <2022-05-01 Sun>

**Comment by AeliaPrima** [comment]
Hello Dr. Wang,

I saw the updated submission date for the project report which is on 2022-05-28 at 8 pm. I wonder if it is our final submission date since it will be after the finals week.

Best,
Yiran

**Comment by Ossifragus** [comment]
Sorry it is a type. I have corrected it to <2022-04-29 Fri>.

**Comment by jguil2022** [comment]
I cannot find a rubric or requirements for the written report.

Where is this posted?

**Reply by Ossifragus** [reply]
https://github.com/STAT5125-UConn/2022Spring/blob/main/project/ProjectScore.md

**Comment by andyslater** [comment]
Is the .md file sufficient for the written report?

**Reply by Ossifragus** [reply]
Yours looks good.

**Comment by andyslater** [comment]
Good morning HaiYing,

 

I believe I have submitted all assignments for the course. Please let me know if there is anything else due.

 

Thank you.

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002

**Reply by Ossifragus** [reply]
No other dues.

**Comment by andyslater** [comment]
Thank you!

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002

**Comment by andyslater** [comment]
Hello Professor Wang,

 

I just wanted to thank you for your class. 

 

I understand the class will be discontinued as-is but I wanted to give a little feedback

*	I appreciated the effort you put into the content, and your discussions in class
*	Gitlab was really nice
*	I enjoyed learning about Julia, although as you know, I think the language goes overboard in some of the syntax
*	Student projects were nice
*	Visualization was cool

 

Some suggestions on what might have added value for me

*	More collaboration with other students – establishing relationships may be as important as the course content
*	More initial discussion on handbook, reference guide and how to find solutions to Julia coding challenges
*	Limit the scope of the language syntax so we could focus more on building simulations
*	More examples of problems and solutions (less language review)

 

In essense, for me, I would like to see the couse focus more on statistical problems and simulation solutions & examples, and less on the programming language syntax

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002

**Reply by Ossifragus** [reply]
Thank you for the feedback.
